framework/CodeInterfaces/PHISICS/combine.py

```diff
@@ -0,0 +1,173 @@
+    paramDict['relapTimePosition'] = relapTimePosition
+    paramDict['depTimeDict'] = depTimeDict
+    paramDict['inpTimeDict'] = inpTimeDict
+    self.joinCSV(paramDict)
+  
+  def findTimePosition(self):
+    """
+      Finds the RELAP time position in the csv file.
+      @ In, None
+      @ Out, timePosition, integer
+    """
+    count = 0
+    with open(self.relapCSV, 'r') as relfile:
+      for line in relfile:
+        count = count + 1
+        for i in xrange(0,len(line.split(','))):
+          if line.strip('\n').split(',')[i] == 'time':
+            return i
+        if count >= 1:
+          raise ValueError("\n the keyword time is not listed in the RELAP csv output \n")
+    
+  def selectPhisicsTime(self,depTimeDict):
+    """
+      Selects the time (<=> line) that will be printed in the final RAVEN output. Those times match with the RELAP one.
+      @ In, depTimeDict, dictionary, information from the xml depletion file
+      @ Out, None
+    """
+    timeList = []
+    self.timeStepSelected = []
+    lineSelected = 0
+    for i in depTimeDict['timeSteps'].split(' '):
+      lineSelected = lineSelected + int(i)
+      self.timeStepSelected.append(lineSelected)
+      lineNumber = 0
+      with open(self.phisicsCSV, 'r') as phifile:
+        for line in phifile:
+          if lineNumber == lineSelected:
+            timeList.append(line.split(',')[0])
+          lineNumber = lineNumber + 1
+    return timeList
+  
+  def putCSVinDict(self,csvFile):
+    """
+      Places each line of the csv into a dictionary. key: line number (integer), value: line (list).
+      @ In, csvFile, string, file name of the csv file parsed
+      @ Out, csvDict, dictionary
+    """
+    countLine = 0
+    csvDict = {}
+    with open(csvFile, 'r') as file:
+      for line in file:
+        if countLine == 0 and csvFile == self.phisicsCSV:
+          self.numOfParameters = self.getNumOfParameters(line)
+        countLine = countLine + 1
+        csvDict[countLine] = line
+    return csvDict
+  
+  def getNumOfParameters(self,line):
+    """
+      Returns the number of parameters that are on one line of the csv file.
+      @ In, line, string, csv file first line
+      @ Out, getNumOfParameters, integer
+    """
+    return len(line.split(','))
+
+  def getNumOfLines(self,csvFile):
+    """
+      Counts the number of lines in the PHISICS or RELAP csv.
+      @ In, csvFile, string, csv file name
+      @ Out, count, integer
+    """
+    count = 0
+    with open(csvFile, 'r') as file:
+      for line in file:
+        count = count + 1
+    return count
+  
+  def joinLine(self,phisicsList,relapList):
+    """
+      Joins the PHISICS and RELAP line into one list.
+      @ In, phisicsList, list, list contianing a csv PHISICS line
+      @ In, relapList, list, list containing a csv RELAP line
+      @ Out, joinedList, list, joined list of parameters
+    """
+    joinedList = []
+    joinedList.append(phisicsList.strip('\n').strip('\r'))
```

Comment:
I do not think a ```list``` has the ```strip``` method. This variables are strings. fIX THE docstrings
And if they are strings this method is quite usefulness
since you can translate it as:
 ```return [phisicsList.rstrip(),relapList.rstrip()]```

Reply:
```rstrip()``` removes ```\n``` and ```\r```